serdes/serde-common/src/main/java/io/apicurio/registry/serdes/SerdeConfigKeys.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.serdes;
```

Comment:
I think we should decide whether we're going to use "serde" or "serdes".  Might be easier to just pick the former and change everything.  What do you think?

Reply:
changed packages to serde, except the root folder, now it is `serdes/serde-common` or `serdes/avro-serde`. The same pattern was applied for the maven module names